### PR TITLE
[VEN-2845]: Add simulations for pufETH support

### PR DESCRIPTION
**Pull request by kkirka**
This PR adds simulations for enabling pufETH market on LST ETH pool

### CODE REVIEW

**Comment by kkirka** [comment]
Sepolia simulation results
image

**Review by chechu — COMMENTED** [review]
Sepolia commands LGTM

File: vips/vip-400/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+  interestRateModel: {
+    address: "0xDaFA3B350288cEb448e0E03077D932f8EF561391",
+    base: "0",
+    multiplier: "0.045",
+    jump: "2",
+    kink: "0.45",
+  },
+  initialSupply: {
+    amount: parseUnits("5", 18),
+    vTokenReceiver: VTREASURY,
```
Comment (chechu) [inline vips/vip-400/bscmainnet.ts:39]:
```suggestion
    vTokenReceiver: "0x495aeBf595D4C641af21A2a021C983C6565CA1A2",
```

Comment (kkirka) [reply vips/vip-400/bscmainnet.ts]: [4c8ee2a](https://github.com/VenusProtocol/vips/pull/401/commits/4c8ee2a3c66e517e8f6bf63157c6cddac3f75930)

File: multisig/proposals/ethereum/vip-100/index.ts
[the remark this answers is not in the document]
Comment (kkirka) [reply multisig/proposals/ethereum/vip-100/index.ts:1]: [74edf3d](https://github.com/VenusProtocol/vips/pull/401/commits/74edf3dc1eebd9c2695ed12a8d5f3c157e6bb56f)

File: multisig/simulations/sepolia/vip-100/index.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      expect(registeredVToken).to.equal(marketSpec.vToken.address);
+    });
+
+    it("check ownership", async () => {
+      expect(await vToken.owner()).to.equal(GUARDIAN);
+    });
+
+    it("check supply", async () => {
+      const expectedSupply = parseUnits("5", 8);
+      expect(await vToken.balanceOf(VTREASURY)).to.equal(expectedSupply);
```
Comment (chechu) [inline multisig/simulations/sepolia/vip-100/index.ts:79]:
```suggestion
      expect(await vToken.balanceOf("0x495aeBf595D4C641af21A2a021C983C6565CA1A2")).to.equal(expectedSupply);
```

Comment (kkirka) [reply multisig/simulations/sepolia/vip-100/index.ts:79]: Fixed here 06cb316565a4f231ecc9b61c4c7606c7e0e1e2ca, simulations pass now 
image

File: hardhat.config.ts [hunk 3]
```diff
@@ -95,12 +95,32 @@ task("test", "Update fork config")
...
 const config: HardhatUserConfig = {
   defaultNetwork: "hardhat",
   networks: {
     hardhat: {
       allowUnlimitedContractSize: true,
       loggingEnabled: false,
+      // Forking networks with unknown hardfork activation history causes errors in
+      // new versions of Hardhat. Following https://github.com/NomicFoundation/hardhat/pull/5394,
+      // we assume Cancun hardfork was active from the beginning for all unknown chains
+      chains: {
```
Comment (chechu) [inline hardhat.config.ts:114]: What if we move this after the `config` definition, so we can reuse the chain ids, with something like this:

```
// Forking networks with unknown hardfork activation history causes errors in
// new versions of Hardhat. Following https://github.com/NomicFoundation/hardhat/pull/5394,
// we assume Cancun hardfork was active from the beginning for all unknown chains
config.networks.hardhat.chains = {
  ...config.networks.hardhat.chains,
  [config.networks.bscmainnet.chainId]: assumeCancun,
  [config.networks.bsctestnet.chainId]: assumeCancun,
  [config.networks.opbnbtestnet.chainId]: assumeCancun,
  [config.networks.opbnbmainnet.chainId]: assumeCancun,
  [config.networks.arbitrumsepolia.chainId]: assumeCancun,
  [config.networks.arbitrumone.chainId]: assumeCancun,
  [config.networks.opsepolia.chainId]: assumeCancun,
  [config.networks.opmainnet.chainId]: assumeCancun,
}
```

**Review by chechu — COMMENTED** [review]
Do we have to change anything at `hardhat.config.zksync.ts`?

File: hardhat.config.ts (hunk 3 above)
Comment (kkirka) [reply hardhat.config.ts:115]: Typescript complains that the inner fields are potentially undefined, but I made it more readable here: [280b164](https://github.com/VenusProtocol/vips/pull/401/commits/280b1643a1a770321f516240b1a0a06f71a0e6b3), lmk if it works

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/395?chainId=56

**Pull request merged**